src/main/python/ayab/engine/options.py

```diff
@@ -128,35 +158,36 @@ def refresh(self):
         # self.ui.color_edit
         # self.ui.start_row_edit
         if self.inf_repeat:
-            self.ui.inf_repeat_checkbox.setCheckState(Qt.Checked)
+            self.ui.inf_repeat_checkbox.setCheckState(Qt.CheckState.Checked)
         else:
-            self.ui.inf_repeat_checkbox.setCheckState(Qt.Unchecked)
+            self.ui.inf_repeat_checkbox.setCheckState(Qt.CheckState.Unchecked)
         # self.ui.start_needle_color
         # self.ui.start_needle_edit
         # self.ui.stop_needle_color
         # self.ui.stop_needle_edit
         self.ui.alignment_combo_box.setCurrentIndex(self.alignment.value)
         if self.auto_mirror:
-            self.ui.auto_mirror_checkbox.setCheckState(Qt.Checked)
+            self.ui.auto_mirror_checkbox.setCheckState(Qt.CheckState.Checked)
             self.ui.auto_mirror_icon.setPixmap(QPixmap(":/garamond-lowercase-e.png"))
         else:
-            self.ui.auto_mirror_checkbox.setCheckState(Qt.Unchecked)
+            self.ui.auto_mirror_checkbox.setCheckState(Qt.CheckState.Unchecked)
             self.ui.auto_mirror_icon.setPixmap(QPixmap(":/garamond-lowercase-e-reversed.png"))
         # self.ui.continuous_reporting_checkbox
 
-    def as_dict(self):
-        return dict([("portname", self.portname),
-                     ("machine", self.machine.name), ("mode", self.mode.name),
-                     ("num_colors", self.num_colors),
-                     ("start_row", self.start_row),
-                     ("inf_repeat", self.inf_repeat),
-                     ("start_needle", self.start_needle),
-                     ("stop_needle", self.stop_needle),
-                     ("alignment", self.alignment.name),
-                     ("auto_mirror", self.auto_mirror),
-                     ("continuous_reporting", self.continuous_reporting)])
-
-    def read(self, portname):
+    def as_dict(self)->OptionsTabDict:
+        return OptionsTabDict({"portname":self.portname,
+                     "machine": self.machine.name, 
+                     "mode": self.mode.name,
+                     "num_colors": self.num_colors,
+                     "start_row": self.start_row,
+                     "inf_repeat": self.inf_repeat,
+                     "start_needle": self.start_needle,
+                     "stop_needle": self.stop_needle,
+                     "alignment": self.alignment.name,
+                     "auto_mirror": self.auto_mirror,
+                     "continuous_reporting": self.continuous_reporting})
+
```

Comment:
This is really only used as a debug printer, so I'm not concerned about changing this.